Makefile

```diff
@@ -69,10 +75,22 @@ libp2p_helper:
 # Alias
 dht: kademlia libp2p_helper
 
-build: git_hooks reformat-diff
+_build/default/src/app/runtime_genesis_ledger/runtime_genesis_ledger.exe:
+	$(info Building runtime_genesis_ledger)
+	ulimit -s 65532 && (ulimit -n 10240 || true) && $(WRAPAPP) env CODA_COMMIT_SHA1=$(GITLONGHASH) dune build --profile=$(DUNE_PROFILE) src/app/runtime_genesis_ledger/runtime_genesis_ledger.exe src/app/runtime_genesis_ledger/genesis_filename.txt
+.PHONY: _build/default/src/app/runtime_genesis_ledger/runtime_genesis_ledger.exe
+
+#Genesis state autogen path
+genesis_path:
```

Comment:
I don't think I really understand this rule. It doesn't seem to do anything.

Reply:
It defines the variable `GENESIS_STATE_PATH` with the filename in `genesis_filename.txt`

Reply:
Actually, it doesn't. The GENESIS_STATE_PATH is on the next line with no indent. Even if there was an indent, it wouldn't create a Makefile variable because it would be interpreted as a shell command and just set an env variable which goes out of scope. Makefile does have the concept of per rule variables which are included, however, they have a different syntax, and the preprocessor expansion would already happen. You Makefile would be equivalent to what it is right now if you delete this `genesis_path:` line.